SixTrack/sixtrack.s

```diff
@@ -61777,11 +61777,21 @@ c$$$            endif
          mat = 6
       elseif (c_material.eq.'C2') then
          mat = 7
+      elseif (c_material.eq.'MoGR') then
+         mat = 8
+      elseif (c_material.eq.'CuCD') then
+         mat = 9
+      elseif (c_material.eq.'Mo') then
+         mat = 10
+      elseif (c_material.eq.'Glid') then
+         mat = 11
+      elseif (c_material.eq.'Iner') then
+         mat = 12
 !02/2008 TW added vacuum and black absorber (was missing) 
       elseif (c_material.eq.'VA') then
```

Comment:
I guessed it was something like that. Since you're anyway modifying, could you please just leave a comment, because it looks a bit weird?